### ISSUE TITLE
Fixing incorrect teaching material and translations

### ISSUE THREAD

**Issue by hubertjb** [NeuroTechX/eeg-101#51]
Going through the app again, I noticed there are a few inconsistencies or incorrect information in the teaching material. I suggest we keep this issue open to flag any such mistakes and then fix them.

In the **English** version:

- "When the eyes are closed, there is often ... The ability to detect alpha waves when the eyes are closed varies greatly from person to person." I don't think 'greatly' is correct, although there is some variation. Maybe look for a reference here.
- "However, when large numbers of cortical neurons fire rhythmically, their activity can produce electric fields that are large enough to cross the surface of the skull." Technically speaking, it always crosses to the surface of the skull. Instead we should say that it's large enough to be picked up there.
- "Thus, data gathered from EEG devices with different reference electrode placement can vary considerably." It's not a matter of devices, but just of where the reference is... E.g., we could re-reference Muse data.
- "They have been associated with alertness, concentration and the active firing of neurons hard at work." I feel like the part about neurons hard at work doesn't mean much. We should maybe rephrase this.
- "USING YOUR MIND TO CONTROL THIS PHONE" It could be a tablet too. :P

In the **translations** (if ever you have a minute @rcassani @neurohazardous @dennlinger :) ):

- In slide 1, the English version used to say the noise introduced by blinks is due to muscle activity. This has been changed to correctly state that noise is caused by eyeball movement. We need to fix the translations.

If you find any other mistake please add them on this issue!

**Comment by neurohazardous** [comment]
@hubertjb Yes, when @rcassani and I did the translation we found some of the things you mention. I am pretty swamped right now, but when you guys correct the english version, please update us so we can do the Spanish translation :dancing_men:

**Comment by jdpigeon** [comment]
The changes have been made and added to the app
